prow/cmd/tot/main.go

```diff
@@ -301,6 +301,7 @@ func main() {
 	}
 
 	http.HandleFunc("/vend/", s.handle)
+	http.HandleFunc("/healthz/ready", func(w http.ResponseWriter, r *http.Request) { fmt.Fprint(w, "OK") })
```

Comment:
Can we please follow the example in `deck` and support both readiness and health endpoints?

https://github.com/kubernetes/test-infra/blob/1f4a1eea2d8aee8b723e274a79c1cb722d091023/prow/cmd/deck/main.go#L161-L163

Reply:
(this is an opportunity to DRY it out as well a la the pprof endpoint)

Reply:
I defined a helper for helath endpoints in `pjutil` package and integrated in deck and tot.